src/Tizen.NUI/src/public/Layouting/LayoutItem.cs

```diff
@@ -584,12 +584,34 @@ private bool SetFrame(float left, float top, float right, float bottom, bool ind
                         // the size of the owner view must be reset even the ExcludeLayouting is true.
                         if (Owner.HeightSpecification < 0 || Owner.WidthSpecification < 0)
                         {
-                            Owner.SetSize(right - left, bottom - top);
+                            if (Owner.WidthResizePolicy == ResizePolicyType.DimensionDependency && Owner.HeightResizePolicy != ResizePolicyType.DimensionDependency)
+                            {
+                                Owner.SetSize(Owner.SizeWidth, bottom - top);
+                            }
+                            else if (Owner.WidthResizePolicy != ResizePolicyType.DimensionDependency && Owner.HeightResizePolicy == ResizePolicyType.DimensionDependency)
+                            {
+                                Owner.SetSize(right - left, Owner.SizeHeight);
```

Comment:
This has same issue.
This SetSize() method may change height resize policy too.
It is still problematic.